stdlib/headernt.c

```diff
@@ -143,26 +145,24 @@ static __inline void __declspec(noreturn) run_runtime(char * runtime,
 #endif
 }
 
-#ifdef __MINGW32__
-int main()
-#else
-void __declspec(noreturn) __cdecl headerentry()
-#endif
```

Comment:
I think the use of a different entry point should have become moot when the mingw port was added, so that's eliminated here by converting all 4 ports to use wmain.